lib/fog/compute/google/models/instance_template.rb

```diff
@@ -0,0 +1,33 @@
+module Fog
+  module Compute
+    class Google
+      class InstanceTemplate < Fog::Model
+        identity :name
+
+        attribute :kind
+        attribute :self_link, :aliases => "selfLink"
+        attribute :description
+        # Properties is a hash describing the templates
+        # @see https://cloud.google.com/compute/docs/reference/rest/v1/instanceTemplates/insert
```

Comment:
Nit: If at all possible, add a minimal hash example. The doc itself is quite complicated and it took me a while to figure the right parameters out (didn't work with IGM api before). For newer people that haven't worked with Google API's at all that may be a higher barrier.

I came up with this as a minimal test case:
```
      :properties => {
          :machine_type => TEST_MACHINE_TYPE,
          :disks => [{
                         :boot => true,
                         :initialize_params =>
                          {:source_image => "projects/ubuntu-os-cloud/global/images/ubuntu-1804-bionic-v20180522"}
                     }],
          :network_interfaces => [{ :network => "global/networks/default" }]
      }
```
, but wanted to check with you since you can probably provide a better real-life example.

Reply:
Done.

No, I'm using template to clone existing template and change the image.